CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ ycm_default_dirs(${VARS_PREFIX})
 option(CODYCO_USES_URDFDOM "Compile yarpWholeBodyInterface with URDF support" TRUE)
 option(COMPILE_AS_SHARED_LIBRARY "Compile ${PROJECT_NAME} as a shared library" TRUE)
 
-option(ENABLE_TESTS "Enable unit testing" FALSE)
+option(YARPWHOLEBODYINTERFACE_ENABLE_TESTS "Enable unit testing" FALSE)
```

Comment:
I think we should "prefix" all the variables in the cmake files so it's easy to locate them in ccmake.

I've just seen that for RPATH I called the project `YARPWBI`. So we should stick with only one prefix.